packages/playwright-ct-react/index.js

```diff
@@ -25,7 +25,10 @@ const plugin = () => {
     path.join(__dirname, 'registerSource.mjs'),
     () => import('@vitejs/plugin-react').then(plugin => plugin.default()));
 };
-const defineConfig = config => originalDefineConfig({ ...config, _plugins: [plugin] });
+const defineConfig = config => {
+  setVitePluginAdded();
```

Comment:
I would mark config as created by defineConfig and would check it on testInfo.config.